index.html

```diff
@@ -320,11 +320,77 @@ <h2 class="food-spin__location-title js-food-spin-location-title">Pick Up </h2>
         </svg>
       </button>
     </section>
+      <div class="cart-summary js-cart-summary" data-open="true">
+        <div class="cart-summary__top">
+          <h2 class="cart-summary__title">My Shopping Cart</h2>
+            <button class="cart-summary__close js-cart-summary-close">
+              <svg class="cart-summary__svg" aria-label="enter store" width="18" height="18" viewBox="0 0 28 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Este dice es `enter store`. No deberia ser `Close cart` o `go back to the store`?